CONTRIBUTING.md

```diff
@@ -138,6 +139,35 @@ When reporting a bug please include the following key pieces of information:
   Submitting a 5 line script will get a much faster response from the team
   than one that's hundreds of lines long.
 
+## Reporting Security Vulnerabilities
+
+If you discover a security vulnerability within KServe, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue for the vulnerability.
+
+2. **DO NOT** disclose the vulnerability publicly until it has been addressed.
+
+3. Report the vulnerability by emailing the KServe security team at security@kserve.io with the following information:
```

Comment:
I don't think we have this email? There's [kserve-security@lists.lfaidata.foundation](mailto:kserve-security@lists.lfaidata.foundation)

Reply:
fixed.